### PR TITLE
Update TensorFlow to 2.3.1

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

**Comment by regadas** [comment]
@nevillelyh we need this one as well https://github.com/nevillelyh/parquet-extra/pull/122

**Comment by nevillelyh** [comment]
@regadas just released 0.4.0

File: build.sbt [hunk 1]
```diff
@@ -804,7 +805,13 @@ lazy val `scio-parquet`: Project = project
     description := "Scio add-on for Parquet",
     libraryDependencies ++= Seq(
       "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
-      "me.lyh" %% "parquet-avro" % parquetExtraVersion,
+      "me.lyh" %% "parquet-avro" % parquetExtraVersion excludeAll (
+        // parquet-avro depends on avro 1.10.x
```
Comment (regadas) [inline build.sbt:809]: @nevillelyh hopefully this will work ok as it doesn't use any of the impact API. 🤞

Comment (nevillelyh) [reply build.sbt:809]: Right I need to make those Avro deps provided. https://github.com/nevillelyh/parquet-extra/issues/123

**Pull request merged**